dbt/models/default/schema.yml

```diff
@@ -21,25 +21,9 @@ models:
             - pin
             - year
       # TODO: Values match what's in whatever iasworld view feeds our website
-  - name: vw_pin_appeal
+  - name: default.vw_pin_appeal
     description: '{{ doc("vw_pin_appeal") }}'
     tests:
-      # Unique by 14-digit PIN and year
-      - unique_combination_of_columns:
-          name: vw_pin_appeal_unique_by_14_digit_pin_and_year
-          combination_of_columns:
-            - pin
-            - year
-          config:
-            error_if: ">280679"
-      # Unique by case number and year
-      - unique_combination_of_columns:
-          name: vw_pin_appeal_unique_by_case_number_and_year
-          combination_of_columns:
-            - year
-            - case_no
-          config:
-            error_if: ">366017"
```

Comment:
Billy said I could delete these, so I went ahead and did that instead of commenting them out like Dan suggested!